README.md

```diff
@@ -38,6 +38,37 @@ Please go through the [Contribution Guidelines](CONTRIBUTING.md) before going fo
 * Run Migrations: `rails db:migrate`
 * At this point, local development can be started with ```rails s -b 127.0.0.1 -p 8080```
 
+### Additional setup instructions for `new_ux`
```

Comment:
I think adding 'new_ux' is not needed cause it is on master now

Reply:
So just 'additional setup instructions'

Reply:
@junron I think it might even be a point of setup instructions list